src/duckstation-qt/mainwindow.cpp

```diff
@@ -814,6 +815,16 @@ void MainWindow::updateEmulationActions(bool starting, bool running)
     }
   }
 
+  if (g_settings.debugging.enable_gdb_server) {
```

Comment:
Probably not the best place for this, but we can move it later if we move the server to the core.